src/users/users.repository.ts

```diff
@@ -44,6 +44,21 @@ export class UsersRepository {
     });
   }
 
+  updateByUsername(
+    username: string,
+    updateUserDto: UpdateUserDto,
+  ): Promise<UserModel> {
+    const message = `UsersRepository.updateByUsername() username=${username} updateUserDto=${JSON.stringify(
+      updateUserDto,
+    )}`;
+    this.logger.log(message);
+    return UserModel.query()
+      .patch({ ...updateUserDto, updatedAt: new Date() })
+      .where('username', username)
+      .returning('*')
+      .first();
+  }
```

Comment:
same reason with `UsersController.updateByUsername`. maybe we can avoid adding this. let me know if it doesn't.